tests/components/homematicip_cloud/conftest.py

```diff
@@ -0,0 +1,82 @@
+"""Initializer helpers for HomematicIP fake server."""
+from unittest.mock import MagicMock, patch
+
+from homematicip.aio.connection import AsyncConnection
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.homematicip_cloud import (
+    DOMAIN as HMIPC_DOMAIN,
+    const as hmipc,
+    hap as hmip_hap,
+)
+from homeassistant.core import HomeAssistant
+
+from .helper import AUTH_TOKEN, HAPID, HomeTemplate
+
+from tests.common import MockConfigEntry, mock_coro
+
+
+@pytest.fixture(name="mock_connection")
+def mock_connection_fixture():
+    """Return a mockked connection."""
+    connection = MagicMock(spec=AsyncConnection)
+
+    def _rest_call_side_effect(path, body=None):
+        return path, body
+
+    def _api_call_side_effect(path, body=None, full_url=False):
+        return True
+
+    connection._restCall.side_effect = _rest_call_side_effect  # pylint: disable=W0212
```

Comment:
FYI We don't run pylint on tests

Reply:
`script/lazytox.py` does do that. It's nice to have clean test runs without warnings.

Reply:
Even if it was not run on test, i personally always use pylint, and i like it clean.